src/SFA.DAS.Admin.Aan.Web/Models/NetworkEvent/EventDescriptionViewModel.cs

```diff
@@ -1,5 +1,5 @@
 ﻿using SFA.DAS.Admin.Aan.Web.Infrastructure;
-using SFA.DAS.Admin.Aan.Web.Validators.ManageEvent;
+using SFA.DAS.Admin.Aan.Web.Validators;
```

Comment:
It feels wrong that a validator is bring referenced in a view model, can the constants be in constants namespace instead of in a validator?

Reply:
yes I can do that